browser/speedreader/speedreader_tab_helper.cc

```diff
@@ -434,6 +438,27 @@ void SpeedreaderTabHelper::OnPrefChanged() {
   UpdateButtonIfNeeded();
 }
 
+void SpeedreaderTabHelper::OnPropertyPrefChanged(const std::string& path) {
```

Comment:
Does this update the current WebContent's the user is on multiple instances of browser window?